### PR TITLE
Test with postgresql and mariadb on multiple versions of Django.

### PR DESCRIPTION
**Pull request by tim-schilling**
Expands the tests for the three latest versions of Django.

* Fixes issues with Django 3.0.0 and postgres.
* Unpins mysqlclient done with https://github.com/jazzband/django-debug-toolbar/commit/995db40a2620210b82e2e5ccde3e2ddbcdeaefcb

### CODE REVIEW

**Comment by tim-schilling** [comment]
I was able to track down the problem to postgres having a different CursorDebugWrapper class defined which was not being monkeypatched by `debugsqlshell`.

File: debug_toolbar/management/commands/debugsqlshell.py [hunk 1]
```diff
@@ -2,12 +2,17 @@
 
 import sqlparse
 from django.core.management.commands.shell import Command  # noqa
-from django.db.backends import utils as db_backends_utils
+from django.db import connection
+
+if connection.vendor == "postgresql":
+    from django.db.backends.postgresql import base as base_module
+else:
+    from django.db.backends import utils as base_module
```
Comment (matthiask) [inline debug_toolbar/management/commands/debugsqlshell.py]: Why is the postgresql branch necessary?

Comment (tim-schilling) [reply debug_toolbar/management/commands/debugsqlshell.py]: Django has a new subclass for the CursorDebugWrapper for postgres. With Django 3 and postgres, the current version of the toolbar's `debug_shell` command doesn't work.

**Comment by tim-schilling** [comment]
You're absolutely right about .travis.yml. For some reason I thought it was using the .tox file.

**Comment by pauloxnet** [comment]
Hi @tim-schilling thanks for your job! I have the same problem described in issue #1231 . Do you have any news about this PR ?

**Comment by gabbork** [comment]
Hi, I have the same issue too. Any news? Thanks a lot

**Comment by filippo-20tab** [comment]
Same issue here. Please, fix this ASAP!

**Comment by tim-schilling** [comment]
Sorry @matthiask I didn't notice the errors. Everything should be good now, unless you want me to change how to handle the conditional import logic.

**Comment by matthiask** [comment]
Awesome, thanks!

**Pull request merged**